cortx-motr.spec.in

```diff
@@ -83,6 +83,7 @@ Provides:       motr-modules = %{version}-%{release}
 
 BuildRequires:  automake
 BuildRequires:  autoconf
+BuildRequires:  autoconf-archive
```

Comment:
Adding any new dependencies must be approved by some CORTX/MOTR committee. @rkothiya @madhavemuri 

Why is this `autoconf-archive` needed?

Reply:
I think it is needed by `AX_CHECK_COMPILE_FLAG` macro.